libdrgn/symbol.h

```diff
@@ -16,6 +16,7 @@ struct drgn_symbol {
 	uint64_t size;
 	enum drgn_symbol_binding binding;
 	enum drgn_symbol_kind kind;
+	enum drgn_lifetime name_lifetime;
```

Comment:
We should add `__attribute__((__packed__))` to the definitions of all of these `enum`s to avoid growing this structure. That can probably be its own preparatory commit for the existing `drgn_symbol_binding` and `drgn_symbol_kind`, then you can include it when you add the attribute in this commit.